metron-analytics/metron-profiler-spark/README.md

```diff
@@ -263,6 +265,18 @@ The path to the input data read by the Batch Profiler.
 
 The format of the input data read by the Batch Profiler.
 
+### `profiler.batch.input.begin`
```

Comment:
Just to be extra clear, is this a > or a >=?

Reply:
I updated the README to clarify this point.